addons/website_slides_survey/views/res_partner_views.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
```

Comment:
to gain one indentation, you could directly do : 
`<odoo><data>`
on the same line

Reply:
+1